src/sweetalert2.js

```diff
@@ -1327,4 +1327,11 @@ sweetAlert.version = ''
 
 sweetAlert.default = sweetAlert
 
+/**
+ * Set default params if `window._swalDefaults` is an object
+ */
+if (typeof window._swalDefaults === 'object') {
```

Comment:
Will this break SSR with window not being defined?

Reply:
> Will this break SSR with window not being defined?

@samturrell Yes, we did it again :nerd_face:   

Some work has been done to prevent such mistakes in future: https://github.com/limonte/sweetalert2/issues/454#issuecomment-353938662